static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx

```diff
@@ -176,66 +175,62 @@ function SimilarStackTrace({params, location, project}: Props) {
           which refers to whether or not we'd group the similar issue into the main issue.
         </Alert>
       )}
-      <Layout.Body>
-        <Layout.Main fullWidth>
```

Comment:
This has been moved to `GroupSimilarIssues`:
<img width="479" alt="image" src="https://github.com/getsentry/sentry/assets/44410/93293870-eba3-4347-a488-ad83d6e85362">